README.md

```diff
@@ -1,4 +1,4 @@
-# AWS Lambda Powertools TypeScript
+# AWS Lambda Powertools (TypeScript)
```

Comment:
I see where the change is coming from since the `README.md` files of the Python & Java versions have it like this:

![image](https://user-images.githubusercontent.com/7353869/148374047-cfcb05c5-91fb-436c-b0e5-ff93d1cfe426.png)

But as you can see from the screenshot above the parenthesis usage is already not consistent (above Python uses it but then two lines below when mentioning Java it's without). Same goes for documentation:

![image](https://user-images.githubusercontent.com/7353869/148374270-6ee0e623-eae7-4d72-b9e0-bd843cb3f11b.png)

I think there's a chance to make this all more consistent in this `README.md` and all other occurrences (docs, other `README.md`, etc.), maybe in a separate PR, so in the spirit of keeping this one focused I'd recommend reverting this change to its original form.

Reply:
It is really just for the github readme, all other places on Java and Python seem to be consistent